recipes/amundsen-databuilder/NOTICE

```diff
@@ -0,0 +1,4 @@
+amundsencommon
+Copyright 2019-2020 Lyft Inc.
```

Comment:
Having this outside of the license text is odd. Makes it hard to interpret if all the code is licensed under Apache-2.0 or not.

Reply:
@ocefpaf  - True, I wasnt sure if I should include that notice, but i checked setup.py on github and https://www.amundsen.io/ both have the license as Apache-2.0.

Reply:
If the license does not require that notice I would not include it. It bring some confusion and make it hard to interpret. Maybe ask upstream about it?

Reply:
removed notice